scripts/leetcode.js

```diff
@@ -219,31 +219,19 @@ function uploadGit(
                 ) {
                   sha = stats.sha[filePath];
                 }
+                const uniqueFileName = `${fileName}-${Math.random()}`;
```

Comment:
This doesn't guarantee the filename will be unique. There is a possibility that the filename could get duplicated due to the randomness.